modules/migration-upgrading-migration-tool-4.adoc

```diff
@@ -4,10 +4,9 @@
 // * migration/migrating_4_2_4/upgrading-migration-tool-4-2-4.adoc
 
 [id='migration-upgrading-migration-tool-4_{context}']
-= Upgrading the {mtc-full} Operator on an {product-title} 4 cluster
+= Upgrading the {mtc-full} on an {product-title} 4 cluster
 
-ifeval::["{mtc-version}" > "1.3"]
-You can upgrade the {mtc-full} ({mtc-short}) Operator on an {product-title} 4 cluster with the Operator Lifecycle Manager.
+You can upgrade the {mtc-full} ({mtc-short}) on an {product-title} 4 cluster by using the Operator Lifecycle Manager (OLM) to install the {mtc-full} Operator.
```

Comment:
@apinnick do we need to clarify the upgrade is from MTC 1.3.x -> 1.4.0? If customer is using 1.2.5, the upgrade guide is not the same.

Reply:
I understood that we are not supporting 1.2 upgrade. Who can clarify this?